doc/api/child_process.md

```diff
@@ -177,6 +177,9 @@ directly by the shell and special characters (vary based on
 [shell](https://en.wikipedia.org/wiki/List_of_command-line_interpreters))
 need to be dealt with accordingly:
 
+If the `shell` option is passed as `false`, then certain shell behaviours such
```

Comment:
```suggestion
If the `shell` option is `false`, then certain shell behaviors such
```

(Our style guide standardizes on US English spellings, so `behaviors` instead of `behaviours`.)

Reply:
The documentation above for the `shell` option says it is a string, but here we are talking about passing a boolean. ????

Reply:
Oh yes, I got your point. When inferring from the code https://github.com/nodejs/node/blob/6107a8f0931a47c92a96f6ea592a4346342276f0/lib/child_process.js#L141 I can see that there is a type check, so passing boolean also will not create an issue and documentation for the same should also be updated.

Reply:
for exec(), the shell defaults to a strong, for execFile(), the shell defaults to a boolean (`false`). I suspect that `true` is not a valid value for options.shell, but have not verified (its definitely not mentioned in the docs).

Reply:
`true` trigger the usage of default parameters, as `false` is a boolean false. Hence the condition for triggering default values is being used by this `true` value, [link](https://github.com/nodejs/node/blob/e080f223b5b21c61806864e4a8f85ccd6fe4b089/lib/child_process.js#L480).